cd/helm/CAPIF/templates/mongo-express-capif-route.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  creationTimestamp: "2022-09-14T08:31:33Z"
+  labels:
+    app.kubernetes.io/managed-by: Helm
+    io.kompose.service: mongo-express
+  name: mongo-express
+  namespace: evol5-capif
+  resourceVersion: "751945857"
+  uid: 6d7963ca-3fd7-41a6-a09a-83bc0bc64bc4
+spec:
+  host: mongo-express.apps.ocp-epg.hi.inet
+  port:
+    targetPort: 8082
```

Comment:
adapt according to mongo-express-service port definition, avoid set port name as a number